### PR TITLE
fix: NFT activity page layout issue

### PR DESCRIPTION
**Pull request by Chef-Yogi**
https://linear.app/pancakeswap/issue/PAN-466/fix-nft-activity-css

### CODE REVIEW

File: src/views/Nft/market/ActivityHistory/ClearAllButton.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const ClearAllButton: React.FC<ClearAllButtonProps> = ({ collectionAddress, ...p
   }
 
   return (
-    <Button key="clear-all" variant="text" scale="sm" onClick={clearAll} {...props}>
+    <Button key="clear-all" variant="text" scale="sm" onClick={clearAll} style={{ whiteSpace: 'nowarp' }} {...props}>
```
Comment (chefilip) [inline src/views/Nft/market/ActivityHistory/ClearAllButton.tsx:19]: Typo? Should be it nowrap?

**Comment by ChefMomota** [comment]
Screenshot 2022-04-29 at 10 54 57 PM

@Chef-Yogi  change `overflow: auto` to `overflow-x: auto`

**Comment by ChefMomota** [comment]
Screenshot 2022-04-29 at 11 01 44 PM

In small device when scroll to right, last bottom border will disappear.

File: src/views/Nft/market/ActivityHistory/ClearAllButton.tsx (hunk 1 above)
Comment (Chef-Yogi) [reply src/views/Nft/market/ActivityHistory/ClearAllButton.tsx:19]: got it! thanks
you catch me 😂

**Comment by Chef-Yogi** [comment]
@ChefMomota  thanks for reminder

**Review by chefilip — DISMISSED** [review]

**Comment by ChefMomota** [comment]
@Chef-Yogi  Ok, thanks. let me open a task for the sec one. LGTM~

**Review by chefilip — APPROVED** [review]

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**